pkg/util/zookeeper_util.go

```diff
@@ -39,15 +49,78 @@ func DeleteAllZnodes(uri string, namespace string, pravegaClusterName string) (e
 			zkSvcPort = zkUri[1]
 		}
 	}
-	hostname := zkSvcName + "." + namespace + ".svc.cluster.local:" + zkSvcPort
-	host := []string{hostname}
+	match := re.MatchString(zkSvcName)
+	if match {
+		hostname = zkSvcName + ":" + zkSvcPort
+	} else {
+		hostname = zkSvcName + "." + namespace + ".svc.cluster.local:" + zkSvcPort
+	}
+	return []string{hostname}
+}
+
+func getRoot(pravegaClusterName string) string {
+	return fmt.Sprintf("/%s/%s", PravegaPath, pravegaClusterName)
+}
+
+func getZnode(pravegaClusterName string) string {
+	return fmt.Sprintf("%s/bookkeeper/conf", getRoot(pravegaClusterName))
+}
+
+func CreateZnode(uri string, namespace string, pravegaClusterName string, replicas int32) (err error) {
```

Comment:
This function and the next do not seem to be utilities to create and update generic znodes. They are adding some concrete data about the cluster. Perhaps name them accordingly for clarity.

Reply:
done